main.go

```diff
@@ -85,8 +122,5 @@ func main() {
 		}
 	})
 
-	log.Infoln("Listening on", *listenAddress)
-	if err := http.ListenAndServe(*listenAddress, nil); err != nil {
-		log.Fatal(err)
-	}
+	return *listenAddress
```

Comment:
Let's make this return custom https://golang.org/pkg/net/http/#NewServeMux instead of listen address